openapi/src/json-schemas/models/definitions/dataset/simple-open-question.json

```diff
@@ -0,0 +1,22 @@
+{
+    "title": "Simple open question",
+    "type": "object",
+    "required": ["type", "id", "label", "closedQuestion", "value"],
+    "properties": {
+        "type": {
+            "const": "simple"
+        },
+        "id": {
+            "$ref": "../question-id.json"
+        },
+        "label": {
+            "type": "string"
+        },
+        "closedQuestion": {
+            "enum": [false]
```

Comment:
could we use `"const": false` here?

Reply:
This is a clunky bit... for some reason speccy won't allow `const` to have a boolean, so it needs to be an enum. (OpenAPI v3.0.2 doesn't support the keyword `const`)

Reply:
yeah thought so. just checking